public/container.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+
+  <title>video player</title>
+  <style type="text/css">
+    video {
+      width: 100%;
+      max-width: 600px;
+      height: auto;
+    }
+  </style>
+
+  <video controls>
+    <source src="https://labs.waterdata.usgs.gov/visualizations/mp4s/what-is-drought.mp4" type="video/mp4">
```

Comment:
Is this hosted locally or through s3?

Reply:
on s3. I think that will work? but we'll see